File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/ContattoAzienda.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class ContattoAzienda
    {
        public List<RisorseUmane> risorseUmane { get; set; }
        public bool contattoPubblico { get; set; }
        public int proprietarioContattoAzienda { get; set; } // Enum_ProprietarioContattoAzienda
        public string aziendaCorrentePIVA { get; set; } // Partita IVA dell'azienda corrente durante la creazione della nuova azienda

    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/ContattoAzienda.ts
import {RisorseUmane} from './RisorseUmane';

export class ContattoAzienda {
  risorseUmane: RisorseUmane[];
  contattoPubblico: boolean;
  proprietarioContattoAzienda: number;
  aziendaCorrentePIVA: string;

  constructor() {
    this.risorseUmane = [];
    this.contattoPubblico = false;
    this.proprietarioContattoAzienda = 0;
    this.aziendaCorrentePIVA = '';
  }
}
